CONTRIBUTING.rst

```diff
@@ -82,3 +82,20 @@ coverage rate should increase or stay the same. Automatic services will ensure
 your code works on all the operative systems and package combinations
 poliastro support - specifically, note that poliastro is a Python 3 only
 library.
+
+Development environment
+-----------------------
+
+To set up a development environment:
+
+1. `Install git <https://git-scm.com/>`_ on your computer.
+2. `Register to GitHub <https://github.com/>`_.
+3. `Fork poliastro <https://help.github.com/articles/fork-a-repo/>`_.
+4. `Clone your fork <https://help.github.com/articles/cloning-a-repository/>`_.
+5. Install it in development mode using
```

Comment:
Would you like to have an environment (either conda's or python's)? That would make you able to test the "stable" versus the "master" and not mess up with other stuff you would have. We [explain conda's way](http://docs.sunpy.org/en/latest/dev_guide/version_control.html#installing-sunpy) in sunpy.